README.md

```diff
@@ -57,10 +57,10 @@ pkg> add ReachabilityAnalysis
 
 ## 📙  Documentation
 
-See the [Reference Manual](https://juliareach.github.io/ReachabilityAnalysis.jl/dev/) for introductory material, examples and API reference.
+See the [Reference Manual](https://juliareach.github.io/ReachabilityAnalysis.jl/dev/) for introductory material, examples and API references.
 
 📌 **Need help?** Have any question, or wish to suggest or ask for a missing feature?
-Do not hesitate to open a [**issue**](https://github.com/JuliaReach/ReachabilityAnalysis.jl/issues) or join the JuliaReach gitter chat: [![join the chat at https://gitter.im/JuliaReach/Lobby](https://badges.gitter.im/JuliaReach/Lobby.svg)](https://gitter.im/JuliaReach/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge), or send an [email](mailto:mforets@gmail.com) to the developers.
+Do not hesitate to open an [**issue**](https://github.com/JuliaReach/ReachabilityAnalysis.jl/issues) or join the JuliaReach Zulip chat: [![join the chat at https://julialang.zulipchat.com](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://julialang.zulipchat.com/#narrow/stream/278609-juliareach). You can also send an [email](mailto:mforets@gmail.com) to the developers.
```

Comment:
maybe change my personal email with `juliareach-dev` one?

Reply:
or otherwise be a bit more explicit with another sentence with the link to the github of main devs (from which the emails and other contact info is easy to find)

Reply:
Since this is independent of this PR, I merge and you can change it separately.